tests/rtm/test_rtm_client.py

```diff
@@ -17,6 +17,14 @@ def setUp(self):
     def tearDown(self):
         slack.RTMClient._callbacks = collections.defaultdict(list)
 
+    def test_run_on_returns_callback(self):
+        @slack.RTMClient.run_on(event="message")
+        def fn_used_elsewhere(**_unusedPayload):
```

Comment:
`_unused_payload` would be better, as it is written with correct casing according to PEP8.